pyveg/src/gee_interface.py

```diff
@@ -90,6 +90,15 @@ def add_NDVI(image, red_band, near_infrared_band):
         print("Something went wrong in the NDVI variable construction")
         return image
 
+def add_count_mosaic(image, RGB_bands):
```

Comment:
Is this function called anywhere? I can't any other reference to it. If not, should it be?

Reply:
Yeah, this was the previous way of creating the COUNT band but that would lead to double counting. It is true that we don't use it anymore and it wasn't worth having a function for the new implementation.  I'll delete the function.